### DISCUSSION TITLE
Unresponsive after update from 2021.1 to 2021.2 using docker

### DISCUSSION THREAD

**Discussion by sswayney** [datalust/seq-tickets#1194] [Help]
Any help would be greatly appreciated. 

We moved from a previous version of image to 2020. Then to 2021.1. Then the update to 2021.2 appears as though it is running but is unresponsive to posts for logs and requests from the ui. I am able to go back to 2021.1 and all is fine. But I am worried. 

We use an .env file, but I don't think that would be an issue since it is the same as it has always been.

# .env
SEQ_API_CANONICALURI=REMOVED_FOR_SEQ_TICKET
BASE_URI=REMOVED_FOR_SEQ_TICKET
HOST_HTTP_PORT=88
HOST_INGESTION_PORT=5341
HOST_PATH_TO_SEQ=/docker/volumes/seq/data
SETTINGS_KEY=REMOVED_FOR_SEQ_TICKET

# docker-compose.yml

version: "2"

services:
seq:
image: 'datalust/seq:2021.2'
restart: always
container_name: seq_log_server
environment:
ACCEPT_EULA: Y
BASE_URI: ${BASE_URI}:${HOST_HTTP_PORT}
SEQ_API_CANONICALURI: ${SEQ_API_CANONICALURI}:${HOST_HTTP_PORT}
volumes:
- $HOST_PATH_TO_SEQ:/data
- ./settings.seqbac:/settings.seqbac
ports:
- $HOST_HTTP_PORT:80
- $HOST_INGESTION_PORT:5341

# Logs:

seq_log_server | ────────────────────────────────────────
seq_log_server | Seq ♦ Machine data, for humans.
seq_log_server | ─────────── © 2021 Datalust Pty Ltd ────
seq_log_server |
seq_log_server | Running as server; press Ctrl+C to exit.
seq_log_server |
seq_log_server | [18:13:55 INF] Seq "2021.2.5647" running on OS "Linux 3.10.0-1160.11.1.el7.x86_64 #1 SMP Fri Dec 18 16:34:56 UTC 2020"
seq_log_server | [18:13:55 INF] Seq detected 33736.536064 MB of RAM
seq_log_server | [18:13:56 INF] Opening event store at "/data/Stream/stream.flare"
seq_log_server | [18:13:56 INF] Opening metastore "/data/Documents/documents.lmdb"
seq_log_server | [18:13:56 INF] Cache warm-up is required
seq_log_server | [18:13:56 INF] Storage subsystem available
seq_log_server | [18:13:56 INF] Seq listening on ["http://localhost/", "http://localhost:5341/"]
seq_log_server | [18:13:56 INF] Seq using canonical URI "REMOVED_FOR_SEQ_TICKET:88/"
seq_log_server | [18:13:56 INF] Started "seqcli" process 135 for "appinstance-515"
seq_log_server | [18:13:56 INF] Started "seqcli" process 137 for "appinstance-139"
seq_log_server | [18:13:56 INF] Started "seqcli" process 136 for "appinstance-421"

Then, nothing.

**Comment by KodrAus** [comment]
Hi @sswayney :wave:


Sorry to hear you've run into trouble.

Do you see any of the regular logs coming through over time about retention policies, metrics sampling, or gen 2 collections? Or is it completely quiet after that point indefinitely?

There are a couple of things we could try:

- Running `docker-compose --verbose up` to see if there are any clues there.
- Exec'ing into the container and running something like `apt-get update; apt-get install -y curl; curl http://localhost/api` and see if you're able to reach the API from within the container itself.
- Exec'ing into the container and using [`dotnet-dump`](https://docs.microsoft.com/en-us/dotnet/core/diagnostics/dotnet-dump) to grab a mini dump of the server. If you do grab one of these you can email it to `support@datalust.co` and we'll take a look.

**Reply by KodrAus** [reply]
Ah if you could share the version of `docker-compose` you're using that might also help narrow things down.
